src/crepe/api/Text.h

```diff
@@ -28,12 +28,6 @@ class Text : public UIObject {
 		 */
 		unsigned int font_size = 16;
```

Comment:
Deze kan weg. Ook graag weghalen in de config.

Reply:
in de config moet bestaan